tools/checkindex.cxx

```diff
@@ -6,9 +6,8 @@
 #include <TFile.h>
 #include <TString.h>
 #include <cstring>
-#include "utils.h"
-
-#include "ramrecord.C"
+#include "../inc/ttree/Utils.h"
```

Comment:
If the tools folder is where the executables are we need to find a better solution to this. The CMakeLists in tools should start depending on `ttree` and then it will need to automatically adjust the include paths so that you do not need to type `../inc`.